### DISCUSSION TITLE
Cannot use import statement outside a module when using internal package

### DISCUSSION THREAD

**Discussion by nooblyf** [vercel/turborepo#3140] [Help]
I followed instructions listed in the docs here to setup an internal schema package:
https://turbo.build/repo/docs/handbook/sharing-code/internal-packages

Everything works fine and I am able to consume my `schema` package in both of my apps but the problem occurs when inside that internal package i try to use import statement, something like `import { z } from 'zod'` i get this error:

```
SyntaxError: Cannot use import statement outside a module
```

**Comment by anthonyshew** [comment]
Do you have a reproduction you could provide?
